api/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -158,6 +158,13 @@ func (in *AWSLoadBalancerSpec) DeepCopyInto(out *AWSLoadBalancerSpec) {
 		*out = new(ClassicELBScheme)
 		**out = **in
 	}
+	if in.SecurityGroups != nil {
+		in, out := &in.SecurityGroups, &out.SecurityGroups
+		*out = make([]SecurityGroup, len(*in))
+		for i := range *in {
+			(*in)[i].DeepCopyInto(&(*out)[i])
+		}
+	}
```

Comment:
Is this change intentional? I wouldn't expect the v1alpha2 generated deep copy to change given the rest of the changes in this PR.

Reply:
You're right, I think I forgot to run `make generate` before pushing